### PR TITLE
remove comment

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by cyphar** [comment]
NACK. There's a very good reason that JSON doesn't support comments. While you might not agree with it, it is a valid reason. Not to mention that the runtime-spec **specifically** tells runtimes to ignore fields they don't recognise (so using `comment` should already work).

**Comment by crosbymichael** [comment]
What does this PR actually do?  I don't understand.

**Comment by wking** [comment]
On Tue, Mar 07, 2017 at 06:06:15AM -0800, Aleksa Sarai wrote:
> NACK. There's a very good reason that JSON doesn't support comments.

This PR is catching the JSON Schema up with [1].  If you don't want
comments in the JSON, you probably want to file a PR removing the Go
field.

[1]: https://github.com/opencontainers/runtime-spec/pull/657/files#diff-7f24d60f0cbb9c433e165467e3d34838R535

**Comment by cyphar** [comment]
-_-

**Comment by crosbymichael** [comment]
I think that just slipped by.  Its not necessary for the spec so we should probably remove it.  The spec should be short and concise, only containing information required for the creation/execution of a container.  

I can see that a comment is a nice to have but not required and probably shouldn't be in the spec.

**Comment by q384566678** [comment]
So the result of your discussion is to delete the `comment` field? Is there a clear rule to delete `comment`?

**Comment by crosbymichael** [comment]
@q384566678 ya, I don't think it should have been added in the first place and just slipped by in review.  We don't have any other  comment fields in the spec.

**Comment by q384566678** [comment]
@crosbymichael As you said, I think  should delete the comment in the spec, as I did in https://github.com/opencontainers/runtime-spec/pull/714/commits/b7a172ddfca13a6657c1f8da506006642ea484dd.

**Comment by cyphar** [comment]
:+1:

File: config-linux.md [hunk 1]
```diff
@@ -549,8 +549,7 @@ Operator Constants:
                    "getcwd",
                    "chmod"
                ],
-               "action": "SCMP_ACT_ERRNO",
-               "comment": "stop exploit x"
+               "action": "SCMP_ACT_ERRN"
```
Comment (hqhq) [inline config-linux.md]: typo?

Comment (q384566678) [inline config-linux.md]: updated, PTAL

**Comment by hqhq** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**